src/PrestaShopBundle/Resources/views/Admin/Component/Layout/search_form.html.twig

```diff
@@ -0,0 +1,66 @@
+        <div class="dropdown-divider"></div>
+        <a class="dropdown-item" data-item="{{ 'Catalog'|trans({}, 'Admin.Global') }}" href="#" data-value="1" data-placeholder="{{ 'Product name, reference, etc.'|trans({}, 'Admin.Navigation.Header') }}" data-icon="icon-book"><i class="material-icons">store_mall_directory</i> {{ 'Catalog'|trans({}, 'Admin.Global') }}</a>
+        <a class="dropdown-item" data-item="{{ 'Customers'|trans({}, 'Admin.Navigation.Header') }} {{ 'by name'|trans }}" href="#" data-value="2" data-placeholder="{{ 'Name'|trans({}, 'Admin.Navigation.Header') }}" data-icon="icon-group"><i class="material-icons">group</i> {{ 'Customers'|trans({}, 'Admin.Navigation.Header') }} {{ 'by name'|trans({}, 'Admin.Navigation.Header') }} </a>
+        <a class="dropdown-item" data-item="{{ 'Customers'|trans({}, 'Admin.Navigation.Header') }} {{ 'by ip address'|trans }}" href="#" data-value="6" data-placeholder="{{ '123.45.67.89'|trans({}, 'Admin.Navigation.Header') }}" data-icon="icon-desktop"><i class="material-icons">desktop_mac</i>{{ 'Customers'|trans({}, 'Admin.Navigation.Header') }} {{ 'by IP address'|trans({}, 'Admin.Navigation.Header') }}</a>
+        <a class="dropdown-item" data-item="{{ 'Orders'|trans({}, 'Admin.Global') }}" href="#" data-value="3" data-placeholder="{{ 'Order ID'|trans({}, 'Admin.Navigation.Header') }}" data-icon="icon-credit-card"><i class="material-icons">shopping_basket</i> {{ 'Orders'|trans({}, 'Admin.Global') }}</a>
+        <a class="dropdown-item" data-item="{{ 'Invoices'|trans({}, 'Admin.Global') }}" href="#" data-value="4" data-placeholder="{{ 'Invoice number'|trans({}, 'Admin.Navigation.Header') }}" data-icon="icon-book"><i class="material-icons">book</i> {{ 'Invoices'|trans({}, 'Admin.Global') }}</a>
+        <a class="dropdown-item" data-item="{{ 'Carts'|trans({}, 'Admin.Global') }}" href="#" data-value="5" data-placeholder="{{ 'Cart ID'|trans({}, 'Admin.Navigation.Header') }}" data-icon="icon-shopping-cart"><i class="material-icons">shopping_cart</i> {{ 'Carts'|trans({}, 'Admin.Global') }}</a>
+        <a class="dropdown-item" data-item="{{ 'Modules'|trans({}, 'Admin.Global') }}" href="#" data-value="7" data-placeholder="{{ 'Module name'|trans({}, 'Admin.Navigation.Header') }}" data-icon="icon-puzzle-piece"><i class="material-icons">extension</i> {{ 'Modules'|trans({}, 'Admin.Global') }}</a>
+      </div>
+      <button class="btn btn-primary" type="submit"><span class="d-none">{{ 'SEARCH'|trans({}, 'Admin.Navigation.Header') }}</span><i class="material-icons">search</i></button>
+    </div>
+  </div>
+</form>
+
+<script type="text/javascript">
+  $(function(){
+    {% if search_type is defined and search_type %}
```

Comment:
SAme here seems like `search_type` should be injected, even if it's not always present in smart global vars

Reply:
nice catch